Src/AutoFakeItEasy2.UnitTest/AutoFakeItEasyCustomizationTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FakeItEasy;
+using Ploeh.AutoFixture.Kernel;
+using Xunit;
+
+namespace Ploeh.AutoFixture.AutoFakeItEasy.UnitTest
```

Comment:
I was on the fence about that. Yes, it's a version specific thing. But, it's still the AutoFakeItEasy thing. I kept the library's namesapce the same, so it wouldn't affect people when they upgrade. This namespace was kept the same to match the library. Whatever your convention is, I'll follow.